src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+body {
+  background: #e5e5e5;
+  margin: 0;
+  min-width: 375px;
+  max-width: 1200px;
```

Comment:
Please check, header must be fullwidth
![image](https://user-images.githubusercontent.com/54936044/228374331-c7651336-0b22-4ab0-a4aa-8c6841460445.png)